hanabi/hanabi/settings.py

```diff
@@ -94,7 +94,7 @@ def get_db_config():
 TEMPLATES = [
     {
         "BACKEND": "django.template.backends.django.DjangoTemplates",
-        "DIRS": [],
+        "DIRS": ["templates"],
```

Comment:
This should be relative to the project's base directory.

```suggestion
        "DIRS": [os.path.join(BASE_DIR, "templates")],
```

Reply:
This directory should be a sibling to `game`, and should contain the base template.